css/style.css

```diff
@@ -26,6 +26,10 @@
   color: #e99941
 }
 
+.card-img-top {
+  max-height: 35%;
```

Comment:
This just squishes the image, I think the way to do this would be with a crop instead.

Reply:
```suggestion
  max-height: 35%;
  object-fit: cover;
```

Reply:
Ok, so after looking at the before and after, it doesn't look like this solution is going to work. All cards are kind of squished.
Here is before.
![image](https://user-images.githubusercontent.com/7327497/105757513-0a368400-5f1c-11eb-9f02-b09f81a04aec.png)
Here is after.
![image](https://user-images.githubusercontent.com/7327497/105757574-20444480-5f1c-11eb-97e3-01c35cdee144.png)

Reply:
I have 2 suggestions which would leave Leth's maps untouched.

### Suggestion 1

```suggestion
  height: 88.88px;
  object-fit: cover;
```

This will squish the other map pics which have different aspect ratios.

Before:

![image](https://user-images.githubusercontent.com/8860322/105760442-7932cd80-5f5a-11eb-87a7-82a01563c583.png)

After:

![image](https://user-images.githubusercontent.com/8860322/105760470-83ed6280-5f5a-11eb-981b-2196a749fd88.png)

### Suggestion 2

```suggestion
  height: 88.88px;
  object-fit: contain;
  background-color: black;
```

This will add a black area on the side of the other map pics which have different aspect ratios.

Before:

![image](https://user-images.githubusercontent.com/8860322/105760604-b8f9b500-5f5a-11eb-9256-5594ac7ce155.png)

After:

![image](https://user-images.githubusercontent.com/8860322/105760653-c7e06780-5f5a-11eb-9280-1b96449c5050.png)